ocf_data_sampler/load/nwp/providers/utils.py

```diff
@@ -41,3 +66,12 @@ def open_zarr_paths(
             **general_kwargs,
         )
     return ds
+
+
+def _tensostore_open_zarr_paths(zarr_path: str | list[str], time_dim: str) -> xr.Dataset:
+    if isinstance(zarr_path, list | tuple):
```

Comment:
```suggestion
    if isinstance(zarr_path, list | tuple) or "*" in str(zarr_path):
```
Do we also want to support "*" wildcards in case there are many files and we don't want to list them all?

Reply:
Yeh sure, I can add this